readme.txt

```diff
@@ -42,4 +42,5 @@ tends to be very limited.
 
 = 1.0.0 =
 
+* February 2018
```

Comment:
We could follow the convention we use in our other plugins and place the date beside the version number, ie:

`= [1.0.0] yyyy-mm-dd =`